src/js/hooks/list-pages/outbound/useOutboundListTableData.jsx

```diff
@@ -132,6 +133,34 @@ const useOutboundListTableData = (filterParams) => {
     });
   };
 
+  const approveRequest = async (id, identifier) => {
+    dispatch(showSpinner());
+    try {
+      await stockMovementApi.updateStatus(id, RequisitionStatus.APPROVED);
+      const successMessage = `You have successfully Approved the request ${identifier}`;
+      Alert.success(successMessage);
+      fireFetchData();
+    } finally {
+      dispatch(hideSpinner());
+    }
+  };
+
+  const rejectRequest = async (id, identifier) => {
+    dispatch(showSpinner());
+    try {
+      await stockMovementApi.updateStatus(id, RequisitionStatus.REJECTED);
+      const successMessage = `You have successfully Rejected the request ${identifier}`;
+      Alert.success(successMessage);
```

Comment:
and a question why don't you use our new notifications?
```js
notification(NotificationType.SUCCESS)({
  message: <message>
})
```

Reply:
I simply used the same Alert as we have in this file

Reply:
because `notification` was implemented later this is why in this file `Alert` was used. I think we should use `notification`, since it lets us do more and this should become a convention. (I know in this case it would evaluate to the same, but just not to mix `Alert` vs `notification`.